packages/core/styles/_reset.scss

```diff
@@ -117,6 +117,13 @@
     box-sizing: border-box;
   }
 
+  .cove .map-title sup,
```

Comment:
@ShawnStringfield - 

The `.cove` line should be removed here. Since that's within `.cdc-open-viz-module` the `.cove` class won't have any affect after migrating to `.cove`. You'll want to move this into the cove package.